.gitignore

```diff
@@ -11,6 +11,7 @@ __pycache__/
 env/
 venv/
 .venv/
+.python-version
```

Comment:
Why was this included?

Reply:
I could have sworn I saw folks discussing adding adding that file to the repo, but it looks like it is not there, adding to ignore might be ok

Reply:
This is used by pyenv to set up a virtual environment. It's similar to a venv directory. Nothing specific to the project, just prevents accidentally committing this to the local repo.

Reply:
I know the purpose of `.python-version`, my point is that it has been included in this repo in the past and I know folks discussed keeping it in the past.

https://github.com/Sceptre/sceptre/tree/20579c49ac62ec720cfedca097bbe55daf825a93
https://github.com/Sceptre/sceptre/search?q=%22.python-version%22&type=commits

Ideally a PR should only touch a limited set of files related to the functionality being added/removed/updated.

Reply:
That's fine. I can pull it out. Not a big deal.

Reply:
This has been pulled out.